sdcm/scylla_bench_thread.py

```diff
@@ -156,7 +156,8 @@ def _run_stress(self, loader, loader_idx, cpu_idx):  # pylint: disable=too-many-
                 "k8s_loader_run_type") == 'dynamic':
             cmd_runner = loader.remoter
             cmd_runner_name = loader.remoter.pod_name
-            cleanup_context = contextlib.nullcontext()
+            cleanup_context = contextlib.ExitStack()
+            cleanup_context.callback(loader.remoter.stop)
```

Comment:
Are you sure we need this commit called "fix(dynamic-loader): make sure dynamic load get stopped" after adding really proper fix for the situation with "reuse_cluster" and "dynamic loaders" combination?

Reply:
I don't know we'll need to check

Reply:
it wasn't needed

remove it.